### PR TITLE
102 funny logs

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

File: R/workflow.R [hunk 1]
```diff
@@ -42,7 +42,7 @@ Workflow <- R6::R6Class(
       dir.create(self$workflowFolder, showWarnings = FALSE)
 
       logWorkflow(
-        message = self$reportingEngineInfo$print(),
+        message = private$reportingEngineInfo$print(),
```
Comment (msevestre) [inline R/workflow.R]: Usually, private member start with a .
so `private$.reportingEngineInfo`

File: tests/dev/script-mean-model-workflow-existing-results.R [hunk 2]
```diff
@@ -0,0 +1,19 @@
+rm(list = ls())
+library(ospsuite.reportingengine)
```
Comment (msevestre) [inline tests/dev/script-mean-model-workflow-existing-results.R]: @pchelle  Simple :)

File: tests/dev/script-mean-model-workflow-existing-results.R [hunk 3]
```diff
@@ -0,0 +1,19 @@
+rm(list = ls())
```
Comment (msevestre) [inline tests/dev/script-mean-model-workflow-existing-results.R]: @pchelle What does this line do? Just curious

File: R/workflow.R (hunk 1 above)
Comment (pchelle) [reply R/workflow.R]: Ok, I'll make sure all these private fields start with "."

File: tests/dev/script-mean-model-workflow-existing-results.R (hunk 3 above)
Comment (pchelle) [reply tests/dev/script-mean-model-workflow-existing-results.R]: This line clears the current working environment.
I use it to make sure that previous tests won't interfere with the current script.
Now that you point this out, it is definitely not necessary for the package and should be removed

**Pull request merged**